sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -834,6 +842,7 @@ export class ManagementClient extends LinkEntity {
           operation: Constants.operations.updateDisposition
         }
       };
+      const associatedLinkName = this._getAssociatedReceiverName(this._context);
```

Comment:
The sessionId from `options.sessionId` should be passed to `this._getAssociatedReceiverName()` here

Reply:
Updated